SlimApp/UI/NewClient/NewClientView.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  NewClientView.swift
+//  SlimApp
+//
+//  Created by Ekaterina Grishina on 11/01/23.
+//
+
+import SwiftUI
+
+struct NewClientView: View {
+    
+    @StateObject var viewModel: NewClientViewModel
+    
+    var body: some View {
+        NavigationView {
+            // TODO: e.grishina rewrite it with ScrollView
+            Form {
+                Section {
+                    TextField("Name", text: $viewModel.name)
+                    TextField("Surname", text: $viewModel.surname)
+                    TextField("E-mail", text: $viewModel.email)
+                        .keyboardType(.emailAddress)
+                    TextField("Phone number", text: $viewModel.phoneNumber)
+                        .keyboardType(.phonePad)
+                }
+                Section {
+                    TextField("Height", text: $viewModel.height)
```

Comment:
Should we use something like a Picker here?

Reply:
For now let's keep it with TextField. I'll come up later with solution how to make picker look nice here